docs/source/tutorials/outliers.ipynb

```diff
@@ -516,10 +544,105 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "937c7e97",
+   "metadata": {},
+   "source": [
+    "## 5. Use cleanlab and `pred_probs` to find outliers in the dataset\n",
+    "\n",
+    "\n",
+    "Another way we can detect unusual exampels in the data is using prediction probabilities for out-of-distribution detection. Instead of using intermediate feature representations, this type of outlier detection simply needs a model's `pred_probs` to work. These methods are faster than using `feature_embeddings` but as a tradeoff perform slightly worse. This is due to the intermediate feature embeddings containing more auxillary information for every example, only some of which is represented in the final predicted probabilites.\n",
+    "\n",
+    "To get `pred_probs` for this example, a Logistic Regression Classifier is fit on the already generated `train_feature_embeddings` from the outlier example above."
```

Comment:
a 4-class Logistic Regression classifier

Reply:
add:  

and the given (in-distribution) label for each training example.  This mimics the standard practice of appending a fine-tuned  output layer at the top of a pretrained neural network to adapt it for a particular classification task (you could do that as well in this tutorial instead of our Logistic Regression, the fine-tuning just may run slower).

Reply:
Just checking you meant a 6 class logistic regression classifier right? 6 animal classes in the trainset

Reply:
yep! sorry apparently I can only count on one hand ...